provisioning/roles/ewallet/tasks/ewallet.yml

```diff
@@ -9,6 +9,8 @@
   command: yarn install
   args:
     chdir: /vagrant/apps/admin_panel/assets
+  register: yarn_result
+  changed_when: "'Already up-to-date' not in yarn_result.stdout"
```

Comment:
Nice one. Probably better update `mix` stuff to always-run and detect changes too (but let's separate PR).